src/main/java/uk/gov/hmcts/reform/prl/documentgenerator/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -120,6 +121,8 @@ public byte[] generateDocument(String templateName, Map<String, Object> placehol
         log.debug("Generate document requested with templateName [{}], placeholders of size[{}]",
             templateName, placeholders.size());
 
+        log.info("template name {} with Case data : {}", templateName ,new Gson().toJson(placeholders));
```

Comment:
not sure, what are you printing here as part of case data. dont print any PII data.